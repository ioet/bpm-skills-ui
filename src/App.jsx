import React from 'react';
import Typography from '@material-ui/core/Typography/Typography';
import AppBar from '@material-ui/core/AppBar/AppBar';
import Toolbar from '@material-ui/core/Toolbar/Toolbar';
import SkillListContainer from './component/container/SkillListContainer';
import ErrorSnackbarContainer from './component/container/ErrorSnackbarContainer';
import Footer from './component/presentational/Footer';
import { AppConst } from './constants';
import FloatingActionButtonContainer from './component/container/FloatingActionButtonContainer';
import DeleteDialogContainer from './component/container/DeleteDialogContainer';

function App() {
  return (
    <div>
      <AppBar position="sticky">
        <Toolbar>
          <Typography variant="h6" color="inherit">
            {AppConst.APP_TITLE}
          </Typography>
        </Toolbar>
      </AppBar>
      <SkillListContainer />
      <Footer />
      <FloatingActionButtonContainer />
      <ErrorSnackbarContainer />
      <DeleteDialogContainer />
    </div>
  );
}

export default App;
