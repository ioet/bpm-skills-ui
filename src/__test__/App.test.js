import React from 'react';
import App from '../App';

describe('<App/>', () => {
  test('Component Renders correctly', () => {
    const appWrapper = shallow(<App />);

    expect(toJson(appWrapper))
      .toMatchSnapshot();
  });
});
