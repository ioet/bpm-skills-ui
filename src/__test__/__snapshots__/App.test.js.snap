// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> Component Renders correctly 1`] = `
<React.Fragment>
  <PageContainer />
</React.Fragment>
`;
